core/backend/src/test/SequentialLogMatcher.ts

```diff
@@ -0,0 +1,117 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { GetMetaDataFunction, IDisposable, Logger, LogLevel } from "@bentley/bentleyjs-core";
+import { IModelHost } from "../IModelHost";
+/**
+ * SequentialLogMatcher match log message in order and remove them from list. Any test using
+ * should check if length come down to zero. If its not then test should fail.
```

Comment:
Missing release tag.

Reply:
Its under /test directory. Do we need to give release tag to test utility?

Reply:
Thanks - I did not notice that.